src/Game.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import PlayScreen from './containers/PlayScreen';
+import TitleScreen from './containers/TitleScreen';
+
+class Game extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      started: false,
+      health: 15,
+      hapiness: {
+        total: 100,
+        hunger: 100,
+        fun: 100,
+        hygiene: 100,
+        sleep: 100,
+      }
+    };
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState.started === false && this.state.started === true) {
```

Comment:
Hey @LidiaYoshino Do you intend to use hooks? If you, maybe useEffect here can be useful :-)

Reply:
I want to work without hooks first, to understand the basics. I'll refactor to add them later or use them in a different project